src/UNL/UCBCN/Event.php

```diff
@@ -122,6 +124,15 @@ function links()
                      'uidcreated'        => 'users:uid',
                      'uidlastupdated'    => 'users:uid');
     }
+
+    public function getStatusWithCalendar(Calendar $calendar) {
+        $che = CalendarHasEvent::getById($calendar->id, $this->id);
```

Comment:
I've never really been a fan of acronyms as variable names, because they are not very readable and tend to not be self documenting.  This method might not be a prime example, but if it was longer and a lot more was going on, I could see myself getting last and wondering 'wait, what is `$che` again?  Naming it `$calendarHasEvent` might be better.